exercises/03_count_byte/03_count_byte_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "testing"
+
+func TestCountByte(t *testing.T) {
+	for _, tc := range []struct {
+		name   string
+		input  string
+		inputb byte
+		want   int
+	}{
+		{"empty", "", 97, 0},
```

Comment:
nit: I recommend avoid using ascii codes directly. For instance `'a'` instead of `97`. `'a'` is also an integral constant in Go, which is exacly `97`, but it's very clear what's that